index.html

```diff
@@ -379,8 +271,56 @@ <h2 class="formTitle">
         </li>
       </ul>
     </footer>
+    <section class="popupMenuBackground alternateDisplay" id="popupMenu">
+      <article class="popupMenuDiv">
+        <img src="./assets/images/popupMenu/Disabled.png" class="popupMenuExitBtn" id="popupMenuExitBtn">
+        <div style="position: relative;">
+          <button  class="popupMenuMobileExitBtn" id="popupMenuMobileExitBtn">
+            <img src="./assets/images/popupMenu/Icon.png">
+          </button>
+          <img  class="popupMobilePic" id="popupMobileImg">
+        </div>
+        <img class="popupMobilePicDesktop" id="popupDesktopImg">
+        <div class="popupMenuTitleDiv">
+          <h3 class="popupMenuTitle" id="popupMenuTitle">Keeping track of hundreds of components</h3>
+          <ul class="greyProjectButtons popupListDesktop popupListDesktopGreen">
+            <li class="LinkButton popupMenubtn" >
+              <a href="" class="LinkButton popupMenubtn" target="_blank" id="seeLiveDesktop">
+                See live
+                <img src="./assets/images/popupMenu/seeLiveBtn.png" class="seeLiveBtn">
+              </a>         
+            </li>
+            <li class="LinkButton  popupMenuB" >
+              <a href="" class="LinkButton  popupMenuB" target="_blank" id="seeSourceDesktop">
+                See Source
+                <img src="./assets/images/popupMenu/seeSourceBtn.png" class="seeSourceBtn">
+              </a>
+            </li>     
+          </ul>
+        </div>
+        <ul class="greyProjectButtons popupListMobile greyList" id="mobileBtnList">
+        </ul>
+        <ul class="greyProjectButtons popupListDesktop greyList" id="desktopBtnList">   
+        </ul>
+        <p class="popupMobileDescription" id="mobilePopupDescription">
+        </p>
+        <ul class="greyProjectButtons popupListMobile ">
+          <li class="LinkButton">
+            <a href=""  class="LinkButton" target="_blank" id="seeLiveMobile">
+              See live
+              <img src="./assets/images/popupMenu/seeLiveBtn.png" class="seeLiveBtn">
+            </a>
+          </li>
+          <li class="LinkButton">
+            <a href="" class="LinkButton" target="_blank" id="seeSourceMobile">
+              See Source
+              <img src="./assets/images/popupMenu/seeSourceBtn.png" class="seeSourceBtn">
+            </a>
+          </li>
+        </ul>
+      </article>
+    </section>
```

Comment:
As per the project requirements, the popup should be created dynamically from the same information which is in the array inside your javascript file.  You are manually displaying the same popup for all the work cards. Kindly use the data from the `works` array to populate your relevant popup.